src/Templates/src/templates/maui-blazor/MauiApp.1.csproj

```diff
@@ -11,7 +11,7 @@
             When specifying both architectures, use the plural <RuntimeIdentifiers> instead of the singular <RuntimeIdentifer>.
             The Mac App Store will NOT accept apps with ONLY maccatalyst-arm64 indicated;
             either BOTH runtimes must be indicated or ONLY macatalyst-x64. -->
-         <!-- ex. <RuntimeIdentifiers>maccatalyst-x64;maccatalyst-arm64</RuntimeIdentifiers> -->
+         <!-- e.g. <RuntimeIdentifiers>maccatalyst-x64;maccatalyst-arm64</RuntimeIdentifiers> -->
```

Comment:
I think doc guidance is to avoid Latin abbreviations, and to use plain English instead:

```suggestion
         <!-- For example: <RuntimeIdentifiers>maccatalyst-x64;maccatalyst-arm64</RuntimeIdentifiers> -->
```

Reply:
Would you happen to have a link to such guidance? 😄

Reply:
Check out the doc guidance site, i.e. https://styleguides.azurewebsites.net/StyleGuide/Read?id=2700&topicid=36396&searchkeyword=latin



😁

Reply:
TIL thanks!